buildSrc/src/main/java/org/opensearch/gradle/testfixtures/TestFixturesPlugin.java

```diff
@@ -171,11 +172,8 @@ public void execute(Task task) {
                 .findFirst();
 
             composeExtension.getExecutable().set(dockerCompose.isPresent() ? dockerCompose.get() : "/usr/bin/docker");
```

Comment:
Should there be a log statement after here to indicate which version of docker compose is being picked?

Reply:
Nit sure to be fair - it will be seen from the command line upon use